src/range_map.rs

```diff
@@ -23,7 +24,7 @@ pub struct RangeMap<T> {
 }
 
 impl<T> RangeMap<T> {
-    /// Create a new RangeMap for the given size, and with the given initial value used for
+    /// Creates a new `RangeMap` for the given size, and with the given initial value used for
```

Comment:
Does some Rust style guide cover this yet? I *strongly* prefer the original code using imperative language.

Reply:
I'm not sure, but the Rust Documentation team officially endorses the present tense. (And so do I, but not that it counts.)

Reply:
Yeah, I suppose we should match the Rust `std` docs.